terraform/modules/forest_node/user-data.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+NEW_USER="sam"
+
+VOLUME_NAME="forest_mainnet_volume"
+
+# Create a login user
+useradd -m "${NEW_USER}"
```

Comment:
```suggestion
useradd --create-home -- "$NEW_USER"
```

Also, why have you gone for `${FOO}` syntax instead of `$FOO`? It's unusual